pkg/hwinfo/hwinfo_nonwindows.go

```diff
@@ -51,14 +80,14 @@ func fetchInventory() (map[string]interface{}, error) {
 		res["baseboard.model"] = reqSys[0].Version
 		res["baseboard.serial_number"] = reqSys[0].SerialNumber
 	} else if err != dmidecode.ErrNotFound {
-		logrus.Errorf("hwinfo: failed fetching baseboard info, %s", err.Error())
+		errs.Addf("failed fetching baseboard info, %s", err.Error())
```

Comment:
Error context gets missing here (e.g. error origin - `hwinfo:`). Maybe leverage https://github.com/pkg/errors to wrap errors with context and return them all in stack?

Reply:
> Error context gets missing here
yip, this is a problem over the whole codebase.
I have this in my mind for some time. Maybe this can be addressed with some focused refactor effort.

Reply:
Wrap(err, "hwinfo") was moved one abstract level higher to hwinfo.go. This is why "hwinfo:" was removed from this place.